qmt/tasks/basic/geometry.py

```diff
@@ -17,7 +17,7 @@ def __init__(self, options=None, name='geometry_1d_task'):
         """
         super(Geometry1D, self).__init__([], options, name)
 
-    def _solve_instance(self, input_result_list, current_options):
+    def _solve_instance(input_result_list, current_options):
```

Comment:
One should probably also mark the derived methods as `@staticmethod`. The decorator is not inherited as far as I understand.